spec/operators/find-spec.ts

```diff
@@ -24,12 +24,6 @@ describe('find operator', () => {
     expectSubscriptions(source.subscriptions).toBe(subs);
   });
 
-  it('should throw if not provided a function', () => {
-    expect(() => {
-      of('yut', 'yee', 'sam').pipe(find('yee' as any));
-    }).to.throw(TypeError, 'predicate is not a function');
-  });
-
```

Comment:
That's correct. The way we were doing these checks was completely incoherent. We'll have a separate discussion about adding these checks at runtime.

One option I thought of was to actually have a second package that wrapped everything with checks like:

```ts
import { map } from 'rxjs-with-runtime-checks/operators'; 
```

(Or something like that, I'm not married to it). But it's something that adds a lot of bloat and not a huge amount of benefit.

Otherwise, we just need to have a single, coherent approach and type check _everything_ in `rxjs` and people will just have to pay for it.